_posts/2019-01-24-how-to-keep-safe-as-a-client.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "how to keep safe as a client"
+date:   2019-01-27 22:00:00 +0800
+categories: blog
+author: Joe Zhang
+---
+
+Recently I am working on a project called transcript, It will use many third party services, generally third party services will expose restful APIs and my project will act as a client.
```

Comment:
You are right, I update my blog and use a virtual project instead of the real name, and remove the image about transcript because actually I don't need to.
Thanks!!